packages/react-components/src/spacing/spacing.scss

```diff
@@ -0,0 +1,16 @@
+:root {
```

Comment:
@kstvsko @martagranisz Are we able to confirm that `spacing` tokens should be `theme-agnostic`? What I mean is if we are sure that there will be no `theme` that could introduce different spacings?

Reply:
For now we assumed spacings are constant for all themes